tools/clang/test/HLSL/max_output_records_duplicate.hlsl

```diff
@@ -0,0 +1,73 @@
+  [MaxRecords(5)] NodeOutput<rec1> Output1,
+  [MaxRecordsSharedWith(Output1)] [MaxRecords(5)] NodeOutput<rec1> Output2, /* expected-error {{only one of MaxRecords or MaxRecordsSharedWith may be specified to the same parameter.}} */ /* expected-note {{conflicting attribute is here}} */ /* expected-note {{conflicting attribute is here}} */
+  [MaxRecords(5)] [MaxRecordsSharedWith(Output1)] NodeOutput<rec1> Output3, /* expected-error {{only one of MaxRecords or MaxRecordsSharedWith may be specified to the same parameter.}} */ /* expected-note {{conflicting attribute is here}} */ /* expected-note {{conflicting attribute is here}} */
+  [MaxRecordsSharedWith(Output6)] [MaxRecords(7)] NodeOutput<rec1> Output4, /* expected-error {{only one of MaxRecords or MaxRecordsSharedWith may be specified to the same parameter.}} */ /* expected-note {{conflicting attribute is here}} */ /* expected-note {{conflicting attribute is here}} */
+  [MaxRecords(7)] [MaxRecordsSharedWith(Output6)] NodeOutput<rec1> Output5, /* expected-error {{only one of MaxRecords or MaxRecordsSharedWith may be specified to the same parameter.}} */ /* expected-note {{conflicting attribute is here}} */ /* expected-note {{conflicting attribute is here}} */
+  [MaxRecords(7)] NodeOutput<rec1> Output6)
+{
+}
+
+[Shader("node")]
+[NodeLaunch("Thread")]
+void DuplicateMax2(
+  RWThreadNodeInputRecord<rec0> InputyMcInputFace,
+  [MaxRecords(1)] NodeOutput<rec1> Output1,
+  [MaxRecordsSharedWith(Output1)] [MaxRecords(2)] NodeOutput<rec1> Output2, /* expected-error {{only one of MaxRecords or MaxRecordsSharedWith may be specified to the same parameter.}} */ /* expected-note {{conflicting attribute is here}} */ /* expected-note {{conflicting attribute is here}} */
+  [MaxRecords(3)] [MaxRecordsSharedWith(Output1)] NodeOutput<rec1> Output3, /* expected-error {{only one of MaxRecords or MaxRecordsSharedWith may be specified to the same parameter.}} */ /* expected-note {{conflicting attribute is here}} */ /* expected-note {{conflicting attribute is here}} */
+  [MaxRecordsSharedWith(Output6)] [MaxRecords(4)] NodeOutput<rec1> Output4, /* expected-error {{only one of MaxRecords or MaxRecordsSharedWith may be specified to the same parameter.}} */ /* expected-note {{conflicting attribute is here}} */ /* expected-note {{conflicting attribute is here}} */
+  [MaxRecords(5)] [MaxRecordsSharedWith(Output6)] NodeOutput<rec1> Output5) /* expected-error {{only one of MaxRecords or MaxRecordsSharedWith may be specified to the same parameter.}} */ /* expected-note {{conflicting attribute is here}} */ /* expected-note {{conflicting attribute is here}} */  
+{
+}
+
+
+void DuplicateMaxSameAttr(
+[MaxRecords(6)] [MaxRecords(3)] NodeOutput<rec1> Output1, /* expected-error {{only one of MaxRecords or MaxRecordsSharedWith may be specified to the same parameter.}} */ /* expected-note {{conflicting attribute is here}} */ /* expected-note {{conflicting attribute is here}} */
```

Comment:
What if `MaxRecords` is specified twice with the same value? That should be okay right?